### PR TITLE
Title casing on ingest.

### PR DESCRIPTION
**Pull request by mccalluc**
- Do we have examples of actual records that need this kind of fix?
- This is applied only to titles which are all lower case... maybe that's not right?
- Capitalization logic itself is questionable.
- Do we have examples where call letter special casing is necessary? How do we distinguish "WILL" from "will"?

### CODE REVIEW

**Comment by mccalluc** [comment]
@caseyedavis12 / @afred : feedback appreciated.

**Comment by caseyedavis12** [comment]
@mccalluc: there are also a lot of records that are in all caps. We would like for those to be title case as well. Example: http://americanarchive.org/catalog/cpb-aacip_16-jw86h4d707

An example of "WILL" call letters in a title is: http://americanarchive.org/catalog/cpb-aacip_16-79v15q57

Other examples where special casing is necessary is anytime the call letters for a station appear in the title. Can you add special logic to overlook call letters for participating stations, as well as NPR, PBS, APTS, APT, NET, APM, PRX, NETA, ITVS, and CPB?

**Comment by mccalluc** [comment]
(Towards #396)

**Comment by mccalluc** [comment]
Note that there is no special processing for "WILL"
- This code is only an ad-hoc clean-up when a title doesn't contain both upper and lower case.
- I think that even in this corpus, use as call letters will be less common that use as a word.
- "US", and "LA", among others, will be cleaned up, but it's easy to construct examples where that is not the right course: I might want to remove them from the huge RE as well.

File: scripts/lib/cleaner.rb [hunk 1]
```diff
@@ -75,6 +75,48 @@ def clean(dirty_xml, name='not specified')
...
+              |WMEB|WNED|WNET|WNYC|WOJB|WOSU|WQED|WQEJ|WRFA|WRNI|WSIU
+              |WTIP|WTIU|WUFT|WUMB|WUNC|WUSF|WVIA|WVIZ|WWOZ|WXXI|WYCC
+              |WYSO|WYSU|YSU)\b/xi) { |match|
+            # Based on:
+            #   ruby -ne '$_.match(/[A-Z]{2,}/){|m| puts m}' config/organizations.yml \
+            #     | grep '[AEIOUY]' | sort | uniq | ruby -ne 'print $_.chop; print "|"'
+            # Removed: AM, AMBER, COLORES, COSI, FETCH, NET, RISE, SAM, STEM, TOLD, WILL
+            match.upcase
+          }
+          .gsub(/\b[^AEIOUY]+\b/i) { |match|
```
Comment (afred) [inline scripts/lib/cleaner.rb]: Does this match any word that does not have a capital vowel? What's the logic here?

Comment (mccalluc) [inline scripts/lib/cleaner.rb]: The `i` flag makes it case insensitive, but yeah, if there's a word without vowels, it's probably an acronym, and it should be all-caps.

Comment (afred) [inline scripts/lib/cleaner.rb]: ah.. missed the `i` flag. Got it.

**Pull request merged**